examples/emodel_pipeline_local_python/ephys_data/.gitignore

```diff
@@ -1 +0,0 @@
-!.gitignore
```

Comment:
if you remove these gitignore files, github will see an empty folder and will remove it from the repo. We can see on the branch that there is no longer an ephys_data, figures or logs folder. Do we know if the example works properly without these folders already inside?

Reply:
Yes, that's correct. The example functions perfectly without these folders. I retained them to maintain consistency with the structure used in BluePyOpt (https://github.com/BlueBrain/BluePyOpt/tree/master/examples/graupnerbrunelstdp) and to allow users to view the folder structures. However, if you believe they are unnecessary, we can certainly remove them.

Reply:
Alright, thanks. Then it's fine like this.